README.rst

```diff
@@ -759,6 +769,26 @@ It sets the number of seconds the notification contents should be stored in the
 If you want cached notification content to never expire, then set it to ``None``.
 Set it to ``0`` if you don't want to store notification contents in cache at all.
 
+``OPENWISP_NOTIFICATIONS_OBJECT_NOTIFICATION_WIDGET``
```

Comment:
The term "object notification" may not be super clear for a user, I'm thinking if we can make this more explicit... what about:

- `OPENWISP_NOTIFICATIONS_IGNORABLE_ADMIN_CLASSES`
- `OPENWISP_NOTIFICATIONS_IGNORABLE_OBJECTS`
- `OPENWISP_NOTIFICATIONS_IGNORE_ENABLED_ADMIN`

Or anything else along these lines? Any better idea?

Reply:
`OPENWISP_NOTIFICATIONS_IGNORE_ENABLED_ADMIN` sounds good. :smile: